### PR TITLE
Set08| Add Node Documentation for...(Documentation Added: 20 nodes)

### PR DESCRIPTION
**Pull request by johnpierson**
### Declarations

Detail Level
Select UV on Face
FamilyParameter.CanAssignFormula
FamilyParameter.Id
FamilyParameter.IsReadOnly
FamilyParameter.IsReporting
FamilyParameter.IsShared
FamilyParameter.Name
FamilyParameter.Unit
GlobalParameter.ByName
GlobalParameter.FindByName
GlobalParameter.Name
GlobalParameter.SetValue
GlobalParameter.Value
GlobalParameter.Visible
Parameter.CreateSharedParameter
Parameter.CreateSharedParameterForAllCategories
Parameter.SharedParameterFile
  
Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [x] Snapshot of UI changes, if any.

### Reviewers
@achintyabhat 

### FYIs
@Amoursol

### CODE REVIEW

File: doc/distrib/NodeHelpFiles/_review/DSRevitNodesUI.DetailLevel.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## In Depth
+`Detail Level` allows for the selection of a view detail level for use in graphic settings.
+
+Available detail levels available include:
```
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/DSRevitNodesUI.DetailLevel.md:4]: Available in this sentence is repetitive. Maybe rephrase as "Detail levels available include:"

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.FamilyParameter.Id.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+## In Depth
+`FamilyParameter.Id` returns an integer representation of the element id of the given parameter.
+
+In the example below, the current Revit document (a door family from the sample model) is open. The parameters in the family are returned, along with the element id as an integer. note, built-In Revit parameters have a negative (-) prefix.
```
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/Revit.Elements.FamilyParameter.Id.md:4]: Change "note, built-In Revit parameters have a negative (-) prefix." to "Note:  Built-In Revit parameters have a negative (-) prefix."

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.GlobalParameter.GroupType.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+## In Depth
+`GlobalParameter.GroupType` returns the group type for the given parameter. Information regarding group types can be found in the documentation located, [here](https://help.autodesk.com/view/RVT/2025/ENU/?guid=Revit_API_Revit_API_Developers_Guide_Introduction_Application_and_Document_Units_html).
```
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/Revit.Elements.GlobalParameter.GroupType.md:2]: Is this documentation link accurate? Not sure how Unit types are relevant here.

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.Parameter.CreateSharedParameter.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+## In Depth
+`Parameter.CreateSharedParameterForAllCategories` will add a parameter with the given values to the given categories in the current Revit document.
```
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/Revit.Elements.Parameter.CreateSharedParameter.md:2]: Isn't the node name just "Parameter.CreateSharedParameter"?

**Review by achintyabhat — CHANGES_REQUESTED** [review]
This set looks good and is almost good to go! A couple of minor comments that needs to be addressed.

**Comment by johnpierson** [comment]
Awesome. All comments fixed.

**Review by achintyabhat — APPROVED** [review]
Thanks for all making all the changes. Looks great!
